resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile

```diff
@@ -27,6 +27,9 @@ ARG spark_uid=185
 # of the Spark distribution. E.g.:
 # docker build -t spark:latest -f kubernetes/dockerfiles/spark/Dockerfile .
 
+RUN echo 'deb https://deb.debian.org/debian buster main' >/etc/apt/sources.list && \
+    echo 'deb-src https://deb.debian.org/debian buster main' >>/etc/apt/sources.list
+
 RUN set -ex && \
```

Comment:
Could you move those new two lines after this line? Then, we can have a single `RUN` command.